cms/static/js/spec/views/course_video_settings_spec.js

```diff
@@ -281,8 +302,286 @@ define(
                 verifyPreferanceErrorState($courseVideoSettingsEl.find('.transcript-languages-wrapper'), false);
             });
 
+            it('shows provider selected view if active provider is present', function() {
+                var $selectedProviderContainerEl = $courseVideoSettingsEl.find('.transcript-provider-wrapper .selected-transcript-provider');
+                expect($selectedProviderContainerEl.find('span').html()).toEqual(courseVideoSettingsView.selectedProvider);
+                expect($selectedProviderContainerEl.find('button.action-change-provider')).toExist();
+                // Verify provider list view is not shown.
+                expect($courseVideoSettingsEl.find('.transcript-provider-wrapper .transcript-provider-group')).not.toExist();
+            });
+
+            it('does not show transcript preferences or organization credentials if N/A provider is saved', function() {
+                var $transcriptProvidersListEl;
+
+                renderCourseVideoSettingsView(null, transcriptionPlans);
+
+                // Check N/A provider
+                changeProvider('');
+
+                $transcriptProvidersListEl = $courseVideoSettingsEl.find('.transcript-provider-wrapper .transcript-provider-group');
+                expect($transcriptProvidersListEl.find('input[type=radio]').length).toEqual(3);
```

Comment:
I think we should also check that `N/A` radio button is selected and all the radio buttons has correct text.